### PR TITLE
Rs 2022.10.05

### PR DESCRIPTION
**Pull request by rschlaikjer**
`>:[`

2022-10-05-191112_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 1]
```diff
@@ -177,32 +186,36 @@ Senpai declares calmly, with a stiff expression on her face.
 [sha:6ceabc4b89f8849fe8b4f1968be63be929966ee1]{
 -- Page 80, Offset 18456.
 -- あまりの無理しているオーラに、思わず顔を見合わせる俺と有彦だった。
-Bewildered by her unreasonable attitude, Arihiko and I instinctively look at each other.
+Arihiko and I shoot glances at each other, bewildered by her unreasonable attitude.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:189]: > Arihiko and I shoot each other a glance

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 2]
```diff
@@ -177,32 +186,36 @@ Senpai declares calmly, with a stiff expression on her face.
...
 }
 [sha:fd50f74a204c1943ba00515d91b7554dd12ab174]{
 -- Page 81, Offset 18457.
 -- 「つまりアレだ。理由は分からんが先輩は遠野をこらしめたいと。つーか土下座レベルで反省させたいと」
-"So that's it, huh? I don't know why, but Senpai wants to punish you. And it ain't no trifling matter. It's at the level that you should be prostrating yourself."
+"So that's it, huh?
+ I dunno why, but looks like Senpai wants to punish you.
+ And I mean %{i}really%{/i} punish, like she's gonna make get down on your knees and beg for forgiveness."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:196]: つまりアレだ is like the beginning of a summarizing.
It's a bit easier to illustrate in English if we don't separate into 2 sentences.

> Not sure why, _but basically it's like_ Senpai wanted to punish you.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 3]
```diff
@@ -177,32 +186,36 @@ Senpai declares calmly, with a stiff expression on her face.
...
 -- Page 82, Offset 18458.
 -- 「い、いえ、そこまでの話じゃないんですけど……」
-"N-no, I don't want you to go that far..."
+"N-No, I wasn't going to go that far..."
 }
 [sha:535c8e0942b2bfd632774f64e2c3ca1419a89068]{
 -- Page 82, Offset 18459.
 -- 「いや、そこまでの話と見た。コイツは人並の情がないんでよく地雷踏むからな。
-"No, it does have to go that far. That guy doesn't have the empathy that an average person has, so he always strikes a nerve.
+"No, it seemed to me like you were gonna.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:206]: > Nah, I'm saying you totally were.

Something about this sentence sounds a bit un-Arihiko-like.
Maybe even reformulate the いや somehow.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 4]
```diff
@@ -177,32 +186,36 @@ Senpai declares calmly, with a stiff expression on her face.
...
+ I mean, this guy doesn't have the empathy that an average person has, so he always ends up putting his foot in it.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:207]: Don't like 'have the empathy that an average person has' + 'putting his foot in it' is UK語.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 5]
```diff
@@ -262,22 +276,27 @@ However...
 [sha:3372298c0e26b47101cb613b38d651b5be82684a]{
 -- Page 88, Offset 18473.
 -- 「おまえ、それ言っていいの俺じゃないか、普通!?」
-"Hey, I should be the one saying that, no?!"
+"Hey, shouldn't I should be the one saying that?!"
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:279]: This sentence is a bit unclear like this. Could use a context verb.

> Hey, shouldn't I be the only one who can strike that kinda deal?!

Or something in that vein.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 6]
```diff
@@ -262,22 +276,27 @@ However...
...
 }
 [sha:3d9424254bc07d6e76b71d1ef40478441f75c765]{
 -- Page 89, Offset 18474.
 -- 「だっておまえバラさねえじゃん。先輩、朝から来てくれたのにおまえは一向にこないし、加えて何かしでかしたんならこれぐらいのリターンねぇとな」
-"You've been caught anyway. Even though Senpai came here this morning, you were nowhere to be seen. And with whatever you've done to her to boot, that's not an unreasonable reaction."
+// I think バラさない here is like "squeal"?
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:284]: Yeah. 'give away [a secret]' or whatever.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 7]
```diff
@@ -262,22 +276,27 @@ However...
...
+"Yeah but there was no way you were gonna.
+ I mean, Senpai's been here all morning, so where the hell were you?
+ If you're guilty of anything else on top of that, this seems like fair penance to me."
 }
 [sha:62eb04e7c685674b553fbb8fdb3395be624106b8]{
 -- Page 90, Offset 18475.
 -- 「いや、でも。だいたい面白い話じゃないだろ、それ」
-"No, but――― it's not a particularly interesting story anyway."
+"No, but―――look, it isn't even an interesting story anyway."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:292]: > it's not even that interesting a story anyway

Is what I would prefer--has a better mouthfeel.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 8]
```diff
@@ -262,22 +276,27 @@ However...
...
 }
 [sha:f051b76da70bdc7dd320f13ef410d1d015714286]{
 -- Page 90, Offset 18476.
 -- 「遠野くんは黙っていてください。さあ乾くん、ハリアップ。こうなったら話を聞くまでテコでも動きません」
-"Tohno-kun, you stay quiet. Now, Inui-kun, hurry up. Now that it's come to this, I won't budge until I hear the story."
+"Tohno-kun, you stay quiet.
+ Okay Inui-kun, hurry up.
+ Now that I know this much, I'm not leaving until I hear the details."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:299]: "I'm not leaving until I hear the details" feels like a bit too generic given its テコでも動きません

>....I won't be deterred until I hear the whole story.

Perhaps?

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 9]
```diff
@@ -292,32 +311,39 @@ Three minutes until homeroom starts.
 [sha:062e1d7bab217e7d38331ebe94a3e46c95eecb75]{
 -- Page 91, Offset 18479.
 -- 先輩とノエル先生が教室で顔を合わせるのはすごく胃が痛くなりそうだし、これはもうホントにしょうがない……。
-It'd be really nerve-wracking if Noel-sensei and Senpai were to meet inside the classroom, so I guess there isn't much of a choice...
+I feel like Senpai and Noel-sensei butting heads in the classroom would be enough to give me an ulcer,
+ so I guess there's nothing for it...
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:315]: I feel like the first clause made the sentence a bit more unclear than it should be.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 10]
```diff
@@ -327,72 +353,79 @@ I understand how she feels.
 [sha:d0a5528802dddce2ba69f579521ee4f65134298f]{
 -- Page 94, Offset 18486.
 -- しかし、人体というものは時としてよく分からない仕組みで動くものなのです。
-However, humans can be swayed in mysterious ways.
+But, the human body just behaves in unpredictable ways sometimes.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:356]: 'mysterious ways' or 'strange ways' feels like the correct collocation

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 11]
```diff
@@ -327,72 +353,79 @@ I understand how she feels.
...
 }
 [sha:be8f1594c88f372ac1217d7258640738c42c9062]{
 -- Page 95, Offset 18487.
 -- 「……その、繋がらない感じがイヤなんです。
-"...Umm, I don't like when things don't connect.
+"Um... I don't like it when things don't connect.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:361]: > ...I don't like the feeling of things not coming together

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 12]
```diff
@@ -327,72 +353,79 @@ I understand how she feels.
...
 }
 [sha:94d0a15621ff0334e76d830559d74dbe8bd5aee1]{
 -- Page 95, Offset 18488.
 -- 誰だってあるでしょ、理由はないけどダメなコトって。食べ物の好き嫌いと同じです」
-Other people are the same, I mean, having things they dislike for some unknown reason. It's the same as liking or disliking some food."
+Everyone's got things that just rub them the wrong way, right?
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:366]: Maybe a bit oversimplified?

> I mean, that goes for everyone, doesn't it? We've all got things that rub us the wrong way.

It's a minor point, admittedly. But trying to use the same colloquial structure here.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 13]
```diff
@@ -327,72 +353,79 @@ I understand how she feels.
...
-...Argh, that's why I said it'd be a boring story, you idiot. Look, Senpai's fed up!"
+// Arihimbo
+Ugh, I told you it'd be a boring story, dumbass.
+ Look, Senpai's totally fed up with this!"
 }
 [sha:6b27da7ef49139af1c668a9f37d314dcc896bad8]{
 -- Page 98, Offset 18493.
 -- 耐えきれなくなって話を逸らす。理由はどうあれ、しりとりに弱いというのはやはり恥ずかしい事だ。
-I couldn't bear it anymore, so I changed the subject. Regardless of the reason, being bad at word chain is pretty embarrassing.
+I can't bear it anymore, so I try and change the subject.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:394]: > Unable to bear it any longer, I try to change the subject.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 14]
```diff
@@ -327,72 +353,79 @@ I understand how she feels.
...
 -- Page 98, Offset 18494.
 -- ……が。先輩はそうは取らなかったようだ。
-...Nevertheless. Senpai doesn't seem to have the same opinion.
+Except... Senpai doesn't seem to share my opinion.
 }
 [sha:792eed9823aab137ed3ff74d7894336f9d4c5802]{
 -- Page 99, Offset 18495.
 -- 「いえ、呆れてなんかいませんよ。逆に感動しました。そんな秘密、遠野くんにもあったんですね」
-"No, I'm not fed up. On the contrary, I'm moved. So you had such a secret too, Tohno-kun."
+"No, I'm not fed up. On the contrary, I'm moved.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:405]: Sentence without object.

> ...On the contrary, it moved me

Or maybe just make it one sentence.

**Review by Petrikow — COMMENTED** [review]
otu

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt (hunk 10 above)
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_1C.txt:356]: mysterious ways

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt (hunk 11 above)
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_1C.txt:361]: best of both worlds
> ...I don't like it when things don't come together

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 15]
```diff
@@ -407,25 +440,28 @@ Our friendly chat with Senpai will have to wait until lunch break.
...
-Well then, Inui-kun, Tohno-kun. Maybe we'll see each other after class if our paths cross. And don't skip class too much, Inui-kun!"
+Well then, Inui-kun, Tohno-kun.
+ Maybe if we're lucky we'll run into each other again after school.
+ And don't skip class too much, Inui-kun!"
 }
 [sha:14912ff1e8f00af6a8960715713425acfa439294]{
 -- Page 103, Offset 18504.
 -- 急いではいるものの自然な足取りで先輩は教室から退出していった。
-Senpai leaves our classroom in a brisk, though very natural pace.
+Senpai leaves our classroom at a brisk but natural pace.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_1C.txt:455]: brisk, but natural

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt (hunk 2 above)
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_1C.txt:196]: Yeah, but 'so that's it, huh?' basically doesn't say anything.

Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_1C.txt:196]: > Alright, I get what's happening here.

?

Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_1C.txt:196]: Yeah, much better.

File: script/Ciel/Day 8/08_01_CIEL08_1C.txt [hunk 16]
```diff
@@ -237,22 +250,23 @@ Ciel-senpai meets my gaze, and then awkwardly looks away.
...
 -- 「もちろん。ただし遠野と仲直りするのが条件っス。遠野の弱味を教えるってコトで手打ちにしてやってください」
-"Of course. But in exchange you'll have to make up with Tohno. I'll tell you his weak point, so please make peace with him."
+"Of course. But on one condition――you've gotta make up with Tohno.
+ I'm telling you his weak point in exchange, seems like a good deal to me."
 }
 [sha:f485a6a4f831e2ea0eb089d400565fe019dc5e78]{
 -- Page 88, Offset 18470.
 -- おおー、と感嘆の声をあげるシエル先輩。
-'Ohhhh', Ciel-senpai says admiringly.
+Ciel-senpai oohs and aahs excitedly.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_1C.txt:263]: i dunno, seems unnatural to me without them but i don't really care that much

**Pull request merged**